README.md

```diff
@@ -1,2 +1,157 @@
+
+:> git checkout -b "new-branch-name"
+
+l) What are some common mistakes that can lead to errors while programming?
+
+typos, off-by-one errors, UI being out of sync with data
+
+m) VStack, HStack, and ZStack are ______stack______ views used for ______layout______.
+
+n) How do you list the branches on your local repository? (Feel free to answer with how you use git, i.e. terminal or another app)
+
+:> git branch -a
+
+o) What happens when @State variable changes in SwiftUI?
+
+The system rerenders the views that depend on data that are bound to the @State var.
+
+p) What is the Single Responsibility Principle?
+
+An architecture principle to organize your code by having separate class, structs, or modules focus on doing only one thing, which makes them easier to compose, refactor, replace, and test.
+
+q) What will the print statement below produce?
+
+var name = “Ozma”
+
+print(“Hello, \(name)!”)
+
+Hello, Ozma
+
+r) What commands can you use in git to download data from a remote repository? What commands can you use in git to send data to a remote repository? (Feel free to answer with how you use git, i.e. terminal or another app)
+
+download data from a remote repository:  git clone, git fetch, git pull
+send data to a remote repository: git push
+
+s) Why is a programming To-Do list important, and what is a minimum viable product?
+
+The To-Do list lets explicitly enumerate the requirements for your project and the implementaion steps necessary to support those requirements. It also allows you to priortize those requirements and steps. A MVP is the minimum set of functionality necessary to demnonstrate the core behavior and value of your product (with out all of the bells and whistles).
+
+t) What is a simple way of describing Binding in SwiftUI?
+
+A binding is a way of connecting a @State property to a UI widget (like a slider) so that it changes when the UI widget's value changes. Similarly, changing the @State property in code will change the UI widget's value.
+
+u) What command do you use in git to move changes from one branch to another? (Feel free to answer with how you use git, i.e. terminal or another app)
+
+git merge
+
+v) What is the type of the variable defined below?
+
+var a = 87
+
+Int
+
+w) What is the difference between var and let?
+
+Variables defined with let are constants and cannot have their values re-assigned. Variables defined with let can have their values re-assigned.
+
+
+## Part 3 Questions
+
+
+As with the short answer questions earlier, try answering these questions from memory before looking for the answer. Some of the fill-in-the-blank questions have a letter before the second blank. This is the first letter of the expected word.
+
+
+In ContentView, lines 1 and 2 show the definition of ____state________ P____property__________.
+
+In ContentView, line 3 shows the definition of a      ____variable________  P____property__________.
```

Comment:
In Swift, this is called a `stored` property. It stores the value of the property somewhere to be used later. It may be a little confusing because the previous lines that have `@State`. `@State` is a special property wrapper for SwiftUI (you won't find it in pure Swift).